### ISSUE TITLE
Integrate Question2Answer to phpFox, not working.

### ISSUE THREAD

**Issue by sanjosedennis7593** [PHPfox-Official/phpfox-v4-issues#1921]
Hi,

I tried to integrate question2answer single sign-in to phpfox, but an error shows that "Call to undefined function redis()", the tutorial i saw was almost 3years ago, is iit because there's a lot of updates happen in phpFox?

![image](https://cloud.githubusercontent.com/assets/16043743/25320090/882a1f28-28d6-11e7-99fb-78d34a80bac8.png)

**Comment by neil96** [comment]
Hi @sanjosedennis7593 ,

Which file do you modify? a core file, a file in module/app/theme or a file does not belong to phpFox?

If you modify a does not belong to phpFox, please remember define PHPFOX_NO_RUN then include file start.php to your file.

Regards.

**Comment by sanjosedennis7593** [comment]
I didn't modify anything in the phpFox codebase.

Okay i'll try it later, i'll keep you updated.

Thanks

**Comment by neil96** [comment]
Hi @sanjosedennis7593 

Is it ok now?

Regards.

**Comment by sanjosedennis7593** [comment]
Not yet, i'll keep you updated when it's done. 

Thanks.

**Comment by PhpFoxJohnJr** [comment]
Hey let me know if you get this done at JohnJr over at phpfox forum.

Phpfox needs something like this...as CeoFox use to have something just like this....but sadly no developer has a current Help system...we have to use a forum for a help system. which is very odd to me.

**Comment by PhpFoxJohnJr** [comment]
This is old but maybe it will help :)

**Comment by neil96** [comment]
@sanjosedennis7593 
Any update?

**Comment by PhpFoxJohnJr** [comment]
Darn...I posted a link when I said does this help but it must have been removed.  I looked at question2answer and it looks like it is going down hill with no support or very little.  I still think we need a help system like tootra help center or CeoFox FAQ's for V3.  Both of those programs actually got it right.  It just too bad neither one works for V4.5 anymore...just when we get a stable version...finally :)
